packages/core-services/src/db/migrations/items/006.trc20Token/index.ts

```diff
@@ -0,0 +1,23 @@
+import { changedCoins, idChanges } from './idChanges';
+
+import {
+  migrateTokenDetailsChangeInDb,
+  migrateTokenIdChangeInDb,
+  resetContractTransactionBlockHeight,
+} from '../../helpers';
+import { IMigrationItem } from '../../types';
+
+const migration: IMigrationItem = {
+  id: '5',
+  name: 'ERC20 token list update migration',
+  up: async db => {
+    await migrateTokenIdChangeInDb(db, idChanges);
+    await migrateTokenDetailsChangeInDb(db, changedCoins);
+    await resetContractTransactionBlockHeight(db, migration.name);
```

Comment:
These functions work only for evm. Do update them for the tron as well. Or generalize them to work for any coin family.

Reply:
Reverting list update for tron.